src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -1143,35 +1132,35 @@ public Models.GetStakingInfoModel GetGetStakingInfoModel()
         }
 
         /// <summary>
-        /// Checks whether the coinstake should be split or not. The rule is that we want to be splitting coins until they are individually around 100 Strats,
-        /// but aiming not to exceed a total amount of coins bigger than (500+1)*3. S
+        /// Checks whether the coinstake should be split or not.
         /// </summary>
         /// <param name="stakedUtxosCount">Number of non-empty UTXOs currently at stake.</param>
         /// <param name="amountStaked">Total amount currently at stake</param>
         /// <param name="coinValue">Value of the coin we are considering to split</param>
         /// <param name="splitFactor">number of time we want to split the coin</param>
```

Comment:
this is a constant, why having it as an argument? In any case, all these parameter comments should be sentences with capital letter start and the dot at the end.